File: Modul 6/Python/PRAK604-2310817220025-FirdaKhoirunisa.py
kode1 = input()
kode2 = input()
bintang = 0
pagar = 0
dimiliki = len(kode1)
diterima = len(kode2)

if dimiliki != diterima:
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(dimiliki):
        if kode1[i] == kode2[i]:
            if kode1[i] == ' ':
                print(" ", end = " ")  
            else:
                print("*", end = " ")
                bintang += 1
        else:
            print("#", end="")
            pagar += 1

    print("\n* =", bintang)
    print("# =", pagar)

    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: Modul 6/C/PRAK604-2310817220025-FirdaKhoirunisa.c
#include <stdio.h>
#include <string.h>

int main(){
    char kode1[50], kode2[50];
    int dimiliki, diterima, i, bintang = 0, pagar = 0;
    scanf("%[^\n]%c", &kode1);
    scanf("%[^\n]%c", &kode2);
    dimiliki = strlen (kode1);
    diterima = strlen (kode2);
    
    if (dimiliki != diterima){
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else{
        for (i = 0; i < dimiliki; i++){
            if (kode1[i] == kode2[i]){
                if (kode1[i] == ' '){
                    printf(" ");
                }
                else{
                    printf("*"); bintang++;
                }
            }
            else{
                printf("#"); pagar++;
            }
        }
        printf("\n* = %d", bintang);
        printf("\n# = %d", pagar);
        if (bintang >= pagar){
            printf("\nPesan Asli\n");
        }
        else {
            printf("\nPesan Palsu\n");
        }
    }
}